src/main/java/org/terasology/core/world/CoreBiome.java

```diff
@@ -50,4 +50,37 @@ public String toString() {
         return this.displayName;
     }
 
+    @Override
+    public float getHumidity() {
+        if (displayName.equals("Ocean") || displayName.equals("Snow")) {
```

Comment:
I don't like the comparison via the display name (it is even up for translation and thus is not guaranteed to match with any of these strings). 
If the temperature is specific to each biome you may want to make it a class property like `displayName` or `id`.